examples/test/canvas-texture/index.html

```diff
@@ -5,49 +5,18 @@
     <title>Canvas Texture</title>
     <meta name="description" content="Canvas Texture - A-Frame">
     <script src="../../../dist/aframe.js"></script>
+    <script src="./components/draw-canvas-rectangles.js"></script>
   </head>
   <body>
-    <script type="text/javascript">
-      AFRAME.registerComponent('update-canvas', {
-        schema: {default: ''},
-
-        init: function () {
-          this.canvas = document.getElementById(this.data);
-          this.ctx = this.canvas.getContext('2d');
-          this.ctx.fillStyle = "rgb(0,0,0)";
-          this.ctx.fillRect(0, 0, this.canvas.width, this.canvas.height);
-        },
-
-        tick: function (t) {
-          let hue = t / 10;
-          this.ctx.fillStyle = 'hsl(' + hue + ', 50%, 80%)';
-          this.ctx.fillRect(0,0,this.canvas.width,this.canvas.height);
-
-          hue = t / 15;
-          this.ctx.fillStyle = 'hsl(' + hue + ', 50%, 60%)';
-          var x = this.canvas.width / 10;
-          var y = this.canvas.height / 10;
-          this.ctx.fillRect(x, y, this.canvas.width - x * 2, this.canvas.height - y * 2)
-
-          hue = t / 20;
-          this.ctx.fillStyle = 'hsl(' + hue + ', 50%, 40%)';
-          x = this.canvas.width / 5;
-          y = this.canvas.height / 5;
-          this.ctx.fillRect(x, y, this.canvas.width - x * 2, this.canvas.height - y * 2);
-        }
-      });
-    </script>
     <a-scene stats>
       <a-assets>
-        <canvas id="helloworldCanvas" crossOrigin="anonymous"></canvas>
+        <canvas id="hello-world-canvas" crossOrigin="anonymous"></canvas>
```

Comment:
If you don't want to spend time discussing it, just leave it as is and let's move on please. It's really disrespectful to just ignore comments and say 'I don't care, let's do it my way'.